vantage6/server/resource/role.py

```diff
@@ -55,7 +56,7 @@ def setup(api, api_base, services):
         RoleRules,
         path + '/<int:id>/rule/<int:rule_id>',
         endpoint='role_rule_with_id',
-        methods=('GET', 'DELETE', 'POST'),
```

Comment:
Any reason why this GET request is not/no longer implemented? It is listed in the API docs so that would be a reason to keep it

Reply:
No, no reason.. Added the GET method back.